src/components/Book.jsx

```diff
@@ -0,0 +1,67 @@
+import styles from '../styles/Book.module.css';
+import progIcon from '../images/progress.jpg';
+
+const Book = () => {
```

Comment:
- [ ] The book data should pass to this component using `props` from the parent component, kindly implement this to use `props` instead